apps/api/src/schema/websoc.ts

```diff
@@ -118,30 +136,69 @@ export const websocQuerySchema = z.object({
       }
       return parsedNums;
     }),
-  instructorName: z.string().optional(),
-  days: daysSchema.optional(),
-  building: z.string().optional(),
-  room: z.string().optional(),
+  instructorName: z.string().optional().openapi({
+    description: "The instructor's name for the course",
+    example: "Anteater, P.",
+  }),
+  days: daysSchema.optional().openapi({}),
+  building: z.string().optional().openapi({
+    description: "The building where the course is held",
+    example: "ALP",
+  }),
+  room: z.string().optional().openapi({
+    description: "The room number where the course is held",
+    example: "1100",
+  }),
   division: z
     .enum(courseLevels)
     .or(z.literal("ANY"))
     .optional()
+    .openapi({
+      description: "Filter courses by course level",
+      example: "LowerDiv",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
   sectionType: z
     .union([z.enum(anyArray), z.enum(websocSectionTypes)])
     .optional()
+    .openapi({
+      description: "Filter courses by section type",
+      example: "Lec",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
   fullCourses: z
     .enum(fullCoursesOptions)
     .optional()
+    .openapi({
+      description: "Filter courses by filled status",
+      example: "SkipFull",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
-  cancelledCourses: z.enum(cancelledCoursesOptions).optional(),
-  units: z.optional(z.literal("VAR").or(z.string())),
-  startTime: timeSchema.optional(),
-  endTime: timeSchema.optional(),
+  cancelledCourses: z.enum(cancelledCoursesOptions).optional().openapi({
+    description: "Include, exclude, or only include cancelled sessions",
+    example: "Exclude",
+  }),
+  units: z.optional(z.literal("VAR").or(z.string())).openapi({
+    description:
+      "Filter courses by unit count. 'VAR' filters for courses with variable unit ranges (e.g., '1-4')",
+    example: "4",
+  }),
+  startTime: timeSchema.optional().openapi({
+    description: "Filter for courses starting at or after this time in 24-hour format",
+    example: "08:00",
+  }),
+  endTime: timeSchema.optional().openapi({
+    description: "Filter for courses ending at or before this time in 24-hour format",
+    example: "17:00",
+  }),
   excludeRestrictionCodes: z
     .string()
     .optional()
+    .openapi({
+      description:
+        "Exclude courses by comma-separated restriction codes, see [Registrar's list](https://www.reg.uci.edu/enrollment/restrict_codes.html)",
```

Comment:
Earlier comment about a different display name for this link stands. You also need to clearly indicate in text that this is a link; look at your local copy of the REST reference to see why the current description is problematic.